src/components/user/repo/account.repo.js

```diff
@@ -13,8 +13,7 @@ const getUserAccount = async (userId, app) => {
 	return account;
```

Comment:
Please do not merge it into feature/SC-7555-DeleteUserTrashbin it must and should work as stand alone.
Only the final trigger is passed later.

Reply:
done